proton

```diff
@@ -1535,9 +1536,17 @@ class Session:
         if "nativevulkanloader" in self.compat_config:
             self.dlloverrides["vulkan-1"] = "n"
 
-        if "enablenvapi" in self.compat_config:
+        if "enablenvapi" in self.compat_config or "forcenvapi" in self.compat_config:
             self.env["DXVK_ENABLE_NVAPI"] = "1"
 
+        if "forcenvapi" in self.compat_config:
+            self.env["DXVK_NVAPI_ALLOW_OTHER_DRIVERS"] = "1"
+            self.env["DXVK_NVAPI_DRIVER_VERSION"] = "99999"
+            self.env["WINE_HIDE_AMD_GPU"] = "1"
+            if "DXVK_CONFIG" not in self.env:
+                self.env["DXVK_CONFIG"] = ";"
+            self.env["DXVK_CONFIG"] = "dxgi.hideAmdGpu = True"
```

Comment:
shouldn't we append `;` if `DXVK_CONFIG` is in `env` and if not set it to empty string and then *append* the `hideAmdGpu` bit or am I misreading this?